.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+script:
+  - cargo build --verbose --all
+  - cargo test --verbose --all
```

Comment:
Huge +1 for this, failing test builds are easy to overlook when doing that last build before pushing and opening a PR.

Reply:
I actually really like travis for how simple it is, but it's limited for larger builds like the firmware. Which means we may need a firmware build server with a jenkins instance :(